third-party/axe/package.json

```diff
@@ -0,0 +1,4 @@
+{
+  "type": "module",
```

Comment:
directory needs a license/readme (or metadata but readme is fine, honestly): go/thirdparty/non-google3